topics/emacs/index.md

```diff
@@ -0,0 +1,13 @@
+---
+topic: emacs
+related: emacs-lisp, vim, spacemacs
+display_name: Emacs
+github_url: https://github.com/emacs-mirror/emacs
+logo: emacs.png
+short_description: An extensible, customizable, free/libre text editor — and more.
+url: https://www.gnu.org/software/emacs/emacs.html
+wikipedia_url: https://en.wikipedia.org/wiki/Emacs
+---
+Emacs is a famously extensible text editor written primarily in Emacs
+Lisp.  While it excels at editing text, Emacs stretches the boundaries
+of what 'text' is.
```

Comment:
think you've got an accidental line break here

Reply:
Where? The text is hard-wrapped; is that a problem?

Reply:
ah- yeah I prefer soft wrap! 🙇

Reply:
Weirdo. Will 'fix' :wink:

Reply:
Fixed.